### PR TITLE
Fix 121 validation update profile yevchuk

### PR DESCRIPTION
**Pull request by r-yevchuk**

### CODE REVIEW

**Comment by vitaliimak** [comment]
@r-yevchuk your task still is in "On hold"

File: client/src/components/profile/info-change.js [hunk 1]
```diff
@@ -113,14 +125,14 @@ class InfoChange extends React.Component {
           <Col md="6">
             <Input
               name="name"
-              placeholder={user.name}
```
Comment (vitaliimak) [inline client/src/components/profile/info-change.js:116]: You should left placeholder with information messages, like "Put your name"

**Review by vitaliimak — CHANGES_REQUESTED** [review]
According to this task https://trello.com/c/kmEV8v69/154-145-fix-3-user-cant-register-password-the-same-as-phone-number-or-email password can't be equal to email too

File: client/src/components/profile/info-change.js (hunk 1 above)
Comment (r-yevchuk) [reply client/src/components/profile/info-change.js:116]: Fixed

**Review by vitaliimak — APPROVED** [review]
Validation works well, but there is another issue
image
I've added task https://trello.com/c/6IrV5Vzz/169-160-fix-updating-profile-information

**Review by vitaliimak — CHANGES_REQUESTED** [review]
I've tried to update password in Postman and it works
image

**Review by vitaliimak — COMMENTED** [review]
I still can use white spaces in start or ending in password
image

**Review by vitaliimak — APPROVED** [review]

**Review by VozNick — APPROVED** [review]

**Review by Melnyk9506 — APPROVED** [review]

**Pull request merged**